choreo-extension-native/src/main/java/io/ballerina/observe/choreo/InitUtils.java

```diff
@@ -40,14 +42,15 @@ private InitUtils() {   // Prevent initialization
     private static final String REPORTER_HOSTNAME_ENV_VAR = "CHOREO_EXT_REPORTER_HOSTNAME";
 
     private static final Logger LOGGER = LogFactory.getLogger();
-
+    private static final BallerinaObserver observer = new StepcountObserver();
```

Comment:
Better to add a new line between fields and methods.
```suggestion
    private static final BallerinaObserver observer = new StepcountObserver();

```